apps/finance/views.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import mixins
+from rest_framework.viewsets import GenericViewSet
+
+from .models import Transaction
+from .serializers import TransactionSerializer
+
+
+class Transactions(mixins.ListModelMixin, mixins.RetrieveModelMixin, GenericViewSet):
```

Comment:
Also only admins should have access to all transaction.. other users should only see their own transactions